modules/claims_api/lib/claims_api/vbms_uploader.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VbmsUploader
+    def initialize(filepath:, file_number:, doc_type:)
+      @filepath = filepath
+      @file_number = file_number
+      @doc_type = doc_type
+    end
+
+    def upload!
+      upload_token_response = fetch_upload_token(
+        filepath: @filepath,
+        file_number: @file_number
+      )
+
+      upload_response = upload_document(
+        filepath: @filepath,
+        upload_token: upload_token_response.upload_token
+      )
+      {
+        vbms_new_document_version_ref_id: upload_response.upload_document_response[:@new_document_version_ref_id],
+        vbms_document_series_ref_id: upload_response.upload_document_response[:@document_series_ref_id]
```

Comment:
the response returns keys that start with `@` ? super weird

Reply:
yes, thats what comes back from VBMS. Very weird but thats the key

Reply:
stop working, Charley

Reply:
lol